llvm-project/build_sparse_libc.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+LLVM_LIBC=$(find . -name libllvmlibc.a)
+for i in strlen memcmp memset memcpy; do
+  llvm-ar x "${LLVM_LIBC}" $i.cpp.o
```

Comment:
Increase indent to four spaces to appease `shfmt`.

Reply:
done